beaker/src/output_manager.rs

```diff
@@ -163,6 +213,9 @@ impl<'a> OutputManager<'a> {
 
         save_metadata(&metadata, &metadata_path)?;
 
+        // Track the metadata file as an output
+        self.track_output(metadata_path.clone());
```

Comment:
don't track the metadata file. Recall

Invariants we want

Depfiles list real byte‑affecting inputs (source, model, tool, config stamp).
Shared TOML is never a prerequisite of earlier targets (later stages may update it).
Idempotent writes: if bytes unchanged, don’t rewrite → mtime stable.
Atomic writes: temp → fsync → rename.
One depfile per target, Make‑escaped, newline‑terminated.